packages/snaps-rpc-methods/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [13.5.1]
+
+### Fixed
+
+- Await `sendWebSocketMessage` promise ([#3608](https://github.com/MetaMask/snaps/pull/3608))
```

Comment:
Should this be a breaking change, since the function return type was changed?

Reply:
The function in question is not exported directly. It is exported as part of a `PermittedHandlerExport` which accepts both async and synchronous implementations: https://github.com/MetaMask/core/blob/2746e05fdcf73cddac9c7ed2f2a808c24fe2afc7/packages/permission-controller/src/utils.ts#L71